Source/Core/DolphinQt/Debugger/MemoryWidget.cpp

```diff
@@ -92,24 +96,46 @@ void MemoryWidget::CreateWidgets()
 
   m_search_address = new QLineEdit;
   m_search_offset = new QLineEdit;
-  m_data_edit = new QLineEdit;
-  m_set_value = new QPushButton(tr("Set &Value"));
 
+  m_search_address->setMaxLength(8);
+  m_search_offset->setMaxLength(9);
   m_search_address->setPlaceholderText(tr("Search Address"));
   m_search_offset->setPlaceholderText(tr("Offset"));
-  m_data_edit->setPlaceholderText(tr("Value"));
 
   m_address_splitter->addWidget(m_search_address);
   m_address_splitter->addWidget(m_search_offset);
   m_address_splitter->setHandleWidth(1);
   m_address_splitter->setCollapsible(0, false);
   m_address_splitter->setStretchFactor(1, 2);
 
-  auto* search_type_group = new QButtonGroup(this);
-  m_find_ascii = new QRadioButton(tr("ASCII"));
-  m_find_hex = new QRadioButton(tr("Hex string"));
-  search_type_group->addButton(m_find_ascii);
-  search_type_group->addButton(m_find_hex);
+  auto* input_layout = new QHBoxLayout;
+  m_data_edit = new QLineEdit;
+  m_base_check = new QCheckBox(tr("Hex"));
```

Comment:
`Parse as Hex` to be consistent with the Cheat Search feature.

Reply:
I wanted to allow as much space for the input box as possible.